src/text_cleaning/text_feature_extractor.py

```diff
@@ -0,0 +1,42 @@
+from functools import reduce
+from typing import Dict, List
+
+import pandas as pd
+from tqdm import tqdm
+
+from src.text_cleaning.spell_checker import SmartSpellChecker
+from src.text_cleaning.utils import (count_how_many_words_are_repeating,
+                                     count_misspelled_words, count_punctuation,
+                                     count_words, preprocess_test)
+
+
+class TextFeatureExtractor:
```

Comment:
clean code comment: All feature extractors we have are text features extractor. This name doesn't clarify the contents of the class. Do you have more informative name?

*Here and further, clean code comments are only for your self-development, and you may discard them if you feel so*